pkgs/applications/networking/browsers/kristall/default.nix

```diff
@@ -1,31 +1,44 @@
-{ lib, mkDerivation, fetchFromGitHub, qtbase, qtmultimedia }:
+{ lib, stdenv, fetchFromGitHub, wrapQtAppsHook, qmake, qtmultimedia }:
 
-mkDerivation rec {
+stdenv.mkDerivation rec {
   pname = "kristall";
   version = "0.3";
+
   src = fetchFromGitHub {
     owner = "MasterQ32";
     repo = "kristall";
-    rev = "V" + version;
+    rev = "V${version}";
     sha256 = "07nf7w6ilzs5g6isnvsmhh4qa1zsprgjyf0zy7rhpx4ikkj8c8zq";
   };
 
-  buildInputs = [ qtbase qtmultimedia ];
+  postPatch = lib.optionalString stdenv.cc.isClang ''
+    sed -i '1i #include <errno.h>' src/browsertab.cpp
+  '';
```

Comment:
not a blocker for this PR, but we should probably create an issue upstream, and then they can use some directives to optionally include this.

Reply:
There are ~400 commits after V0.3, it maybe already fixed...